pydantic/schema.py

```diff
@@ -587,7 +589,9 @@ def field_singleton_sub_fields_schema(
     (EmailStr, {'type': 'string', 'format': 'email'}),
     (UrlStr, {'type': 'string', 'format': 'uri'}),
     (DSN, {'type': 'string', 'format': 'dsn'}),
+    (SecretStr, {'type': 'string', 'writeOnly': True, 'format': 'secret-str'}),
     (str, {'type': 'string'}),
+    (SecretBytes, {'type': 'string', 'writeOnly': True, 'format': 'secret-bytes'}),
```

Comment:
Here again, I would suggest removing the `'format': 'secret-str'` and leaving only the `'writeOnly': True`.

The difference between one `format` and the other is that one is for bytes and the other for strings. But JSON is a text based representation format, it doesn't have a native bytes-like type. So, it would have to still be a string (text). In fact, to send bytes via JSON they would have to be encoded in text (e.g. Base64).